renovate.json5

```diff
@@ -0,0 +1,14 @@
+{
+  extends: ["github>netlify/renovate-config:default"],
+  ignorePresets: [":prHourlyLimit2"],
+  semanticCommits: true,
+  masterIssue: true,
+  packageRules: [
+    {
+      // update Netlify packages immediately
+      packagePatterns: ["^@netlify", "^netlify"],
+      groupName: "netlify packages",
+      schedule: null,
+    },
+  ],
```

Comment:
why is this rule not part of our default config?

Reply:
Good point. See https://github.com/netlify/renovate-config/pull/7